.azure/gpu-tests.yml

```diff
@@ -73,7 +73,7 @@ jobs:
         CUDA_VERSION_MM=$(python -c "import torch ; print(''.join(map(str, torch.version.cuda.split('.')[:2])))")
         pip install "bagua-cuda$CUDA_VERSION_MM>=0.9.0"
         pip install -e .[strategies]
-        pip install deepspeed==0.6.4  # TODO: remove when docker images are upgraded
+        pip install deepspeed>0.6.4  # TODO: remove when docker images are upgraded
```

Comment:
@carmocca the GPU CI passed. Should we address this TODO now or leave it like this?

Reply:
I would leave it like this. When the PR is merged and the dockers get rebuilt overnight, we can remove it without fear of things breaking

Reply:
Alternatively, if we remove it now, even though we validated that this worked inside the PR, there's a window (from the time of merge to the docker rebuilds) where things could go bananas :banana:

Reply:
Ok. I was just wondering if there was an action to take before merge. 
I like bananas though.